pkg/toolchains/BUILD

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# toolchain to wrap an rpmbuild binary
+
+load("//toolchains:rpmbuild.bzl", "rpmbuild_toolchain", "is_rpmbuild_available")
+
+filegroup(
+    name = "standard_package",
+    srcs = glob([
+        "*",
+    ]),
+    visibility = ["//distro:__pkg__"],
+)
+
+# Expose the availabilty of an actual rpmbuild as a feature flag, so  we can
+# select() on it.
+is_rpmbuild_available(
+    name = "is_rpmbuild_available",
+    visibility = ["//:__pkg__"],
+)
+
+toolchain_type(
+    name = "rpmbuild_toolchain_type",
+    visibility = ["//visibility:public"],
+)
+
+# This is an example for the future.
+#rpmbuild_toolchain(
+#    name = "system_rpmbuild",
+#    path = "/usr/bin/rpmbuild",
+#)
+#
+#toolchain(
+#    name = "rpmbuild_linux_toolchain",
+#    exec_compatible_with = [
+#        "@platforms//os:linux",
+#    ],
+#    target_compatible_with = [
+#        "@platforms//os:linux",
+#    ],
+#    toolchain = ":system_rpmbuild",
+#    toolchain_type = ":rpmbuild_toolchain_type",
+#)
```

Comment:
This should probably be stashed along with the documentation.  Fine for now, though.

Reply:
This is WIP. I'm not sure it makes any sense here.

Reply:
I built an example section which uses this PTAL